cmd/platform/deployment-template/pull.go

```diff
@@ -42,4 +45,5 @@ func init() {
 	Command.AddCommand(pullCmd)
 	pullCmd.Flags().StringP("path", "p", "", "Local path where to store deployment templates")
 	pullCmd.MarkFlagRequired("path")
+	pullCmd.Flags().String(format, "deployment", "If deployment is specified deployment_template is populated in the response, If cluster is specified cluster_template is populated in the response. (Defaults to deployment)")
```

Comment:
```suggestion
	pullCmd.Flags().String(format, "deployment", "If a deployment is specified, the deployment_template is populated in the response. If a cluster is specified, the cluster_template is populated in the response. (Defaults to deployment.)")
```

(Similar comment as higher up.)